tap_mambu/schemas/loan_transactions.json

```diff
@@ -129,7 +157,7 @@
       ],
       "additionalProperties": false,
       "properties": {
-        "tax_interest_from_arrears_amount": {
+        "tax_on_interest_from_arrears_amount": {
```

Comment:
I this a fix of a typo? Curious if users of the tap have ever gotten data for this field. If not, then this change is okay, otherwise it'd be backwards incompatible and folks would have to re-sync their data to populate it.

Reply:
You're right, from what I can tell on our end, it was a typo, and no data should have ever been extracted, but for backwards compatibility's sake I will just duplicate the field (so it's an addition, not a modification) and we'll remove it in the next PR (with the rest of the breaking changes